docs/governance/community-guidelines/content-standards-and-practices/identifiers.md

```diff
@@ -0,0 +1,100 @@
+# CCC Taxonomy ID Assignment Guidelines
+
+This document provides a standardized approach for assigning unique identifiers (IDs) to various entities within the CCC Taxonomy, including service types, service categories, features, threats, controls, test requirements, and tests. Adhering to this standard ensures consistency and clarity across all related documentation and implementations.
+
+| **ID Type**          | **Format**                                 | **Example**           |
+|----------------------|--------------------------------------------|-----------------------|
+| **Service Type**     | `CCC.<ServiceType>`                        | `CCC.Storage`         |
+| **Service Category** | `CCC.<ServiceCategory>`                    | `CCC.ObjStor`         |
+| **Feature**          | `CCC.<ServiceCategory>.F<##>`              | `CCC.ObjStor.F01`     |
+| **Threat**           | `CCC.<ServiceCategory>.T<##>`              | `CCC.RDMS.T01`        |
+| **Control**          | `CCC.<ServiceCategory>.C<##>`              | `CCC.VM.C01`          |
+| **Test Requirement** | `CCC.<ServiceCategory>.C<##>.TR<##>`       | `CCC.VM.C01.TR01`     |
+| **Test**             | `CCC.<ServiceCategory>.C<##>.TR<##>.T<##>` | `CCC.VM.C01.TR01.T01` |
+
+## Service Types
+
+Service types are used to group related service categories under a common label.
+
+Each service type ID follows the format `CCC.<ServiceType>`.
+
+### Examples:
+
+- **CCC.Storage** - Represents the Storage service type.
+- **CCC.Compute** - Represents the Compute service type.
+- **CCC.DB** - Represents the Database service type.
+- **CCC.Network** - Represents the Networking service type.
+
+## Service Categories
+
+Service categories are specific classifications within a service type. Because the service category name is unique, the ID does not need to reference the parent type.
```

Comment:
Will this cause confusion between what are Service Types / Service Categories? Would we lose some information on the hierarchy? Would this make it harder to navigate/search?

(Just looking for a discussion on these to make sure they have been considered, rather than saying we can't take this approach for these reasons, as I appreciate that it including the service type could just add a lot of noise).

Reply:
This hasn't been a significant discussion to-date. Happy to hear additional feedback. My primary motivation with this was to reduce the ID length by a few characters, as it's significant once we get down to the lower level entries.

Reply:
Hi @smendis-scottlogic  - in a working session @eddie-knight and I discussed the pros/cons of concatenating service types and categories. Our conclusion was to not change and leave them as-is as proposed.